### ISSUE TITLE
[filterscripts] stop using `omkey.om` (Trac #2019)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2011]
There are places in filterscripts that still make a decision about icetop vs inice by using the `omkey.om` number.  Stop doing this, and start using the geometry to tell what type of dom this is.

Migrated from https://code.icecube.wisc.edu/ticket/2019
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:15:18", 
    "description": "There are places in filterscripts that still make a decision about icetop vs inice by using the `omkey.om` number.  Stop doing this, and start using the geometry to tell what type of dom this is.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067318169976", 
    "component": "combo reconstruction", 
    "summary": "[filterscripts] stop using `omkey.om`", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2017-05-11T16:44:24", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20180613T17:44:27: 
* olivas changed type from "defect" to "cleanup"
* olivas changed milestone from "December 2017" to "Autumnal Equinox 2018"

**Comment by icecube-trac** [comment]
Trac update at 20180922T12:22:13: 
* olivas changed status from "new" to "assigned"
* olivas changed owner from "nwandkowsky" to "olivas"

**Comment by icecube-trac** [comment]
Trac update at 20181001T21:40:51: 
* olivas changed status from "assigned" to "closed"
* olivas commented:

> In changeset 165893:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="165893"
> early preparation for the upgrade.  don't select omtype on om numbers, but the actual omtype, which requires access to the geometry.  closes #2011.
> ```


* olivas changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190112T03:51:14: nwhitehorn commented:

> In changeset 170544:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170544"
> early preparation for the upgrade.  don't select omtype on om numbers, but the actual omtype, which requires access to the geometry.  closes #2011.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:15:18: 
* olivas changed milestone from "Autumnal Equinox 2018" to ""
* olivas commented:

> Milestone Autumnal Equinox 2018 deleted
